README.md

```diff
@@ -99,5 +102,21 @@ interface CrossoverService
 | EvenBitsCrossoverServiceImpl      | even bits copied from g2      | even bits copied from g1      |
 | BitPairCrossoverServiceImpl       | even bit pairs copied from g2 | even bit pairs copied from g1 |
 
+<p>MutatorService, an interface responsible for mutating gene values. Based on given strategies selected bits chenging its value in order to
+    faster find target. </p>
+<p>Classes that implement MutatorService have additional mutationChance field that is set in setter (takes random value from 0 - 100) and represents probability of mutation in percent. Set in seter becouse in each generation of Gene mutationChance can be different</p>
```

Comment:
I would argue that float [0-1) is more natural representation of 0-1 range number (0-100%).

Reply:
you have right, so I will change it fo float